components/datatypes/data/order.schema.json

```diff
@@ -64,6 +71,46 @@
           },
           "meta:titleId": "order##xdm:orderType##title##56861",
           "meta:descriptionId": "order##xdm:orderType##description##62731"
+        },
+        "xdm:lastUpdatedDate": {
+          "title": "Last Updated Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The time when a particular order record is last updated in the commerce system.",
+          "meta:titleId": "order##xdm:lastUpdatedDate##title##86152",
+          "meta:descriptionId": "order##xdm:lastUpdatedDate##description##47783"
+        },
+        "xdm:createdDate": {
+          "title": "Created Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The time/date when a new order is created in the commerce system.",
+          "meta:titleId": "order##xdm:createdDate##title##86153",
+          "meta:descriptionId": "order##xdm:createdDate##description##47783"
+        },
+        "xdm:shipDate": {
+          "title": "Ship Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The date when one or more items from an order is shipped. ",
+          "meta:titleId": "order##xdm:shipDate##title##86154",
+          "meta:descriptionId": "order##xdm:shipDate##description##47784"
+        },
+        "xdm:returnDate": {
```

Comment:
In traditional commerce systems, returns are generally modeled outside orders as they are pretty significant to have its own entity. We do not have modeled returns in AEP yet so, I would suggest to first of all create a proposition in a WIKI page to add "returns" properties in AEP. We can go through it and will figure out the best way to model it,

It might be modeled as its own data type and added to its own "returns" object rather than adding it under the same orders. We can discuss further.

Reply:
I can put up a proposal together for introducing a returns datatype into the mix. If returns need to be independent of order as an object/datatype by itself, that should work for us too.